hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -18,12 +18,7 @@
 
             Stimulus:
               - Enable spi_host ip
-              - Clear/enable interrupt (if needed)
-              - Program configuration registers (CONTROL) to reset spi_fsm, reset/set watermark for the fifos
-              - Program the CONFIGOPTS register to config spi_host channel
-                operating in different modes (single, dual, or quad)
-              - Randomize the content of COMMAND register
-              - Program/retrive data in TXDATA/RXDATA register w.r.t transmitted/received transactions
+              - Write data in standard mode - and read it back
```

Comment:
Is this just abbreviating the steps on the left?

Reply:
@martin-lueker it is simplifying them.
the first test should just be very basic.  So for V1 this will only support standard mode an a minimum of randomization,